action.yml

```
@@ -29,6 +29,17 @@ inputs:
   docker-token:
     description: 'Docker Hub token.'
     required: true
+  extra-requirements:
+    description: 'Private repository to be cloned.'
+    required: false
+    default: 'false'
```

shouldn't it be `false`?. @sergivalero20 
```suggestion
    default: false
```

---

I tried to change it before, but I got this error: _**A string representing the default value. The default value is used when an input parameter isn't specified in a workflow file.**_. However I can change the default value to ''. @Jacatove